README.md

```diff
@@ -21,7 +21,7 @@ debugging, testing, and running your training and inference models with DataRobo
 This repository addresses the DataRobot functionality known as `custom models`. The terms `custom model` and `user model` can be used interchangeably, as can `custom model directory` and `code directory`.
 
 ## Quickstart <a name="quickstart"></a>
-The following example shows how to use the [DRUM](https://github.com/datarobot/datarobot-user-models/tree/master/custom_model_runner) tool to make predictions on an [sklearn regression model](model_templates/python3_sklearn). For the training model quickstart, please reference [this document](QUICKSTART-FOR-TASKS.md)
+The following example shows how to use the [DRUM](https://github.com/datarobot/datarobot-user-models/tree/master/custom_model_runner) tool to make predictions on an [sklearn regression model](model_templates/python3_sklearn). 
```

Comment:
two things:
- while you're here, can you fix the line spacing to something close to 100 characters, please.
- and when i test your new link in pycharm it doesn't redirect.  have you tested it with a doc renderer?